sectionproperties/analysis/cross_section.py

```diff
@@ -834,16 +819,14 @@ def calc_plastic():
             plastic_section = PlasticSection(self.geometry, debug)
             plastic_section.calculate_plastic_properties(self, verbose)
 
-        if time_info:
+        if self.time_info:
             text = "--Calculating plastic properties...\n"
```

Comment:
To be consistent with other time outputs remove the addition of the new line at the end `\n`?

Reply:
I think I should just change the other outputs :) They are printing an empty string to create a new line. A new line character is more explicit. You are right, they should be consistent!

Reply:
Agreed. Done!